src/expr/impl/src/udf/mod.rs

```diff
@@ -22,14 +22,14 @@ use risingwave_expr::sig::{
     CreateFunctionOutput, CreateOptions, UDF_IMPLS, UdfImpl, UdfImplDescriptor,
 };
 
-#[cfg(feature = "external-udf")]
+#[cfg(feature = "udf")]
 #[cfg(not(madsim))]
 mod external;
-#[cfg(feature = "python-udf")]
+#[cfg(feature = "udf")]
 mod python;
-#[cfg(feature = "js-udf")]
+#[cfg(feature = "udf")]
 mod quickjs;
-#[cfg(feature = "wasm-udf")]
+#[cfg(feature = "udf")]
 mod wasm;
```

Comment:
Do you mean we want to lift the support to all UDFs to the same level? Otherwise maybe separate feature flags are still good?

Reply:
What do you mean by "lift the support to all UDFs to the same level"? IIUC currently we always build all UDF languages, I don't think we may have some day only build JS UDF support or something like that.

Reply:
not for production, but may only want to develop some of the udfs. (reduce compile time)

Reply:
I think it's the same rationale for adding flags for each connectors

Reply:
That's exactly my motivation. We have plenty of optional modules that are not related to the development of other modules. I don't think it's that necessary to split UDF feature into parts. From my experience, when I want to develop UDF related things, I always want to enable all the features.